sdk/confidentialledger/Azure.Security.CodeTransparency/samples/Sample3_UseYourCredentials.md

```diff
@@ -0,0 +1,18 @@
+# Use your credentials
+
+Your service might have [additional configuration][CTS_configuration_doc] which requires you to pass the authentication token in addition to the signature envelope.
+
+Use [DefaultAzureCredential][default_cred_ref] to get your token:
+
+```C#
+var credential = new DefaultAzureCredential();
```

Comment:
minor nits:

- We should consider staying consistent between the forms:
  `var name = new Type(...);`   and  `Type name = new(...)`

- Guidelines discourage `var` for return values like GetToken.  We should update these patterns to be :  `TokenCredential token = credential.GetToken(...);`   _(as much as I dislike that, it would be good to be consistent)_

Reply:
We are not using Entra/AAD internally, the token extraction is really necessary. I think what you refer to is the usual generated code that is capable of handling the `TokenCredential` but it does not exist in our case.

Reply:
What I'm referring to is that some parts of the samples use the form:

```csharp
SomeType thing = new();
```

While others use the form:

```csharp
var thing = new SomeType();
```

Either form is fine, but the sample use should be consistent.